clang/runtime/dpct-rt/include/atomic.hpp.inc

```diff
@@ -383,6 +383,37 @@ inline T atomic_fetch_max(T *addr, T operand,
 }
 // DPCT_LABEL_END
 
+// DPCT_LABEL_BEGIN|atomic_fetch_compare_dec|dpct
+// DPCT_DEPENDENCY_EMPTY
+// DPCT_CODE
+/// Atomically decrease the value stored in \p addr if old value stored in \p addr
+/// is equal to zero or greater than \p operand, else set 0 to the value stored in \p addr.
```

Comment:
the commnet is not correct.  Doesn't align with the code.

Reply:
Will align it in the updated PR.